Include/cpython/genobject.h

```diff
@@ -46,6 +44,7 @@ PyAPI_FUNC(PyObject *) PyGen_NewWithQualName(PyFrameObject *,
 PyAPI_FUNC(int) _PyGen_SetStopIterationValue(PyObject *);
 PyAPI_FUNC(int) _PyGen_FetchStopIterationValue(PyObject **);
 PyAPI_FUNC(void) _PyGen_Finalize(PyObject *self);
+PyAPI_FUNC(PyCodeObject *) PyGen_GetCode(PyGenObject *gen);
```

Comment:
While I can see the *potential* general utility of this function, does it make sense to add it to the *public* API right now?  (No one has ever asked for it before.  How would someone have gotten this info before?  Through getattr on the the Python object?)  For me it would be enough if you anticipate it would be useful enough and a stable feature, which it essentially is.

If there isn't a solid reason to add this to the public API then I'd expect the function to be "private" (leading underscore) and, maybe, for the name to reflect that it only exists for testing.

Reply:
The problem is that if there is no accessor function, but the field appears to match the Python attribute, C programmers just access the field directly (usually Cython, but there are others).

We can't really complain about it if there is no API function for them to use.

Reply:
Yeah, makes sense.